app/views/foreman/job_templates/install_errata_-_katello_ansible_default.erb

```diff
@@ -15,12 +15,10 @@ kind: job_template
 %>
 
 <% if @host.operatingsystem.family == 'Suse' -%>
-<% advisories = input(:errata).split(',').join(' ') -%>
-<%= render_template('Run Command - Ansible Default', :command => "zypper -n install -t patch #{advisories}") -%>
-<% elsif @host.operatingsystem.family == 'Debian' -%>
-<% advisories = input(:errata).split(',').map { |e| @host.debs_for_erratum(e) }.join(' ') -%>
```

Comment:
Unfortunately, this is not in katello upstream yet - there is another PR to add errata support for Debian/Ubuntu